020-Connect/070-Rust/010-sqlx.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Connect to Postgres with rust sqlx
+navTitle: sqlx
+keywords: ['rust', 'postgres', 'sqlx']
```

Comment:
The keywords are mainly for our in-site search, so 'postgres' might pollute other searches.

```suggestion
keywords: ['rust', 'sqlx']
```